en/docs/install-and-setup/install-and-setup-overview.md

```diff
@@ -0,0 +1,955 @@
+                    </li>                     		    
+ 			    <li>
+ 			        <a>Working with Secrets</a>
+ 			    </li>
+                    <li>
+                        <a href="{{base_path}}/install-and-setup/setup/mi-setup/security/encrypting_plain_text">Encrypting Secrets using WSO2 Secure Vault</a>
+                    </li>
+                    <li>
+                        <a href="{{base_path}}/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets">Using Hashicorp Secrets</a>
+                    </li>
+                    <li>
+                        <a href="{{base_path}}/install-and-setup/setup/mi-setup/security/single_key_encryption">Using Symmetric Encryption</a>
+                    </li>                                          			    
+ 			    <li>
+ 			        <a href="{{base_path}}/install-and-setup/setup/mi-setup/security/securing_management_api">Securing the Management API</a>
+ 			    </li>
+ 			    <li>
+ 			        <a href="{{base_path}}/install-and-setup/setup/mi-setup/customizing_error_pages">Customizing Error Messages</a>
+ 			    </li> 			    		     			     			    
+ 			</ul>
+ 		</td>
+ 	</tr>
+ 	<tr>
+ 		<th>
+ 			<a>Performance</a>
+ 		</th>
+ 		<td>
+ 			Explains how to configure the Micro Integrator at different levels to optimize performance.
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/tuning_jvm_performance">Tuning JVM Performance</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/network_os_performance">Tuning Network and OS Performance</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/jdbc_tuning">Tuning JDBC Configurations</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/http_transport_tuninge">Tuning the HTTP Transport</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/jms_transport_tuning">Tuning the JMS Transport</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/tuning-the-VFS-Transport">Tuning the VFS Transport</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/rabbitmq_transport_tuning">Tuning the RabbitMQ Transport</a>
+            </li>
+            <li>
+                <a href="{{base_path}}/install-and-setup/setup/mi-setup/performance_tuning/tuning-inbound-endpoints">Tuning the Inbound Endpoints</a>
+            </li>
+ 		</td>
+ 	</tr>
+ 	<tr>
+ 		<th>
+ 			<a href="{{base_path}}/install-and-setup/install/installing-the-product/installing-the-binary/installing-on-windows">Message Brokers</a>
```

Comment:
This link seems incorrect. Please check and fix.

Reply:
Removed the link.